README.md

```diff
@@ -107,7 +107,16 @@ Follow the steps below to set up a testnet smart wallet using `starkli`:
 
 ### Ethereum smart contract
 
-First, the Ethereum smart contracts must be deployed. For Ethereum the deployment process you will need to:
+First, make sure you have the necessary dependencies installed in the `contracts/solidity/lib` folder.
+You can install these commands by executing them within the `/contracts/solidity/` folder.
```

Comment:
I would rethink this phrase for better clarity. Which commands should i run to install which dependencies?

Reply:
makes sense, i just updated this